doc/source/io.rst

```diff
@@ -440,6 +440,64 @@ individual columns:
     Specifying ``dtype`` with ``engine`` other than 'c' raises a
     ``ValueError``.
 
+.. note::
+
+   Reading in data with columns containing mixed dtypes and relying
+   on ``pandas`` to infer them is not recommended. In doing so, the
```

Comment:
This is way too negative. you are  describing a very rare situation, where very odd data is passed. make this clear.